src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -8,6 +8,8 @@ import * as React from 'react';
 import debounce from 'lodash/debounce';
 import noop from 'lodash/noop';
 import flow from 'lodash/flow';
+import type { TaskType, TaskNew } from './activity-feed/task-new/flowTypes';
```

Comment:
can we keep type imports at the bottom

Reply:
This doesn't seem to be accurate based on a spot check of the components in this repo. I'm seeing something more like:

```
npm imports
npm type imports
local imports
local type imports
style imports

type defs
```

Reply:
I guess I swapped the type and styles.